doc/classes/Translation.xml

```diff
@@ -88,6 +102,14 @@
 				The number [param n] is the number or quantity of the plural object. It will be used to guide the translation system to fetch the correct plural form for the selected language.
 			</description>
 		</method>
+		<method name="get_plural_messages" qualifiers="const">
+			<return type="PackedStringArray" />
+			<param index="0" name="src_message" type="StringName" />
+			<param index="1" name="context" type="StringName" default="&amp;&quot;&quot;" />
+			<description>
+				Returns an array of message's translation plural forms.
```

Comment:
```suggestion
				Returns an array of plural forms associated to the given [param src_message].
```
or
```suggestion
				Returns an array of plural forms derived from the given [param src_message].
```
or similar.
It's a bit unclear what "plural form" is here, although I *kind of* get it.